cylc/flow/scripts/cat_log.py

```diff
@@ -139,7 +139,7 @@ def colorise_cat_log(cat_proc, color=False):
     if color:
         color_proc = Popen(
             [
-                'python3', '-c',
+                sys.executable, '-c',
```

Comment:
Yep, that works :+1:

Reply:
Nice, note to self to ensure we use `sys.executable` when performing Python subprocess calls.

Reply:
@oliver-sanders  and @hjoliver and @dpmatthews I've realised that I might be able to make tests run without Activating conda by creating a `cylc get-python-exec` CLI, allowing the tests to invoke the correct python.

Is this a good idea? Perhaps it's for a follow on?